docs/api.rst

```diff
@@ -1032,6 +1032,7 @@ Setup or remove an exchange
    :reqjson string passphrase: An optional passphrase, only for exchanges, like coinbase pro, which need a passphrase.
    :reqjson string kraken_account_type: An optional setting for kraken. The type of the user's kraken account. Valid values are "starter", "intermediate" and "pro".
    :reqjson list binance_markets: An optional setting for binance and binanceus. A list of string for markets that should be queried.
+   :reqjson string ftx_subaccount: An optional setting for FTX. This sets the subaccount that will be used in the instance of FTX.
```

Comment:
Add an explanation here. I am not sure what exactly a subaccount is from my tests.